Model/Cron.php

```diff
@@ -247,34 +247,36 @@ public function processNotification()
 
         foreach ($notifications as $notification) {
             // set Cron processing to true
-            $notification->setProcessing(true);
-            $notification->setUpdatedAt(new \DateTime());
-            $notification->save();
+            $this->_updateNotification($notification, true, false);
         }
 
         // loop over the notifications
         $count = 0;
         foreach ($notifications as $notification) {
 
-            $this->_adyenLogger->addAdyenNotificationCronjob(
-                sprintf("Processing notification %s", $notification->getEntityId())
```

Comment:
Don't we still need to log the entityID?
Or are we doing it somewhere else?

Reply:
we log the entity id of the notification so you can trace what specific notification is being processed.